packages/gatsby-source-contentful/src/create-schema-customization.js

```diff
@@ -96,9 +96,39 @@ export async function createSchemaCustomization(
     }),
   ]
 
-  const { getGatsbyImageFieldConfig } = await import(
-    `gatsby-plugin-image/graphql-utils`
-  )
+  let getGatsbyImageFieldConfig = null
+  try {
+    const graphqlUtils = await import(`gatsby-plugin-image/graphql-utils`)
+    getGatsbyImageFieldConfig = graphqlUtils.getGatsbyImageFieldConfig
+  } catch (err) {
+    const isYarn = process.env.npm_config_user_agent.includes(`yarn/`)
+
+    let command = `npm install gatsby-plugin-image`
+    if (isYarn) {
+      command = `yarn add gatsby-plugin-image`
```

Comment:
I don't think other error messages care about this so I'm leaning a bit towards removing this as our docs stance is also do only ever show npm (as with that you'd then also need to support pnpm etc.)